Neos.ContentRepository.Core/Classes/Feature/NodeDuplication/Dto/NodeSubtreeSnapshot.php

```diff
@@ -105,8 +110,26 @@ public static function fromArray(array $array): self
             isset($array['nodeName']) ? NodeName::fromString($array['nodeName']) : null,
             NodeAggregateClassification::from($array['nodeAggregateClassification']),
             SerializedPropertyValues::fromArray($array['propertyValues']),
-            NodeReferencesSnapshot::fromArray($array['nodeReferences']),
+            SerializedNodeReferences::fromArray($array['nodeReferences']),
             $childNodes
         );
     }
+
+    private static function serialiseGraphReferences(References $references): SerializedNodeReferences
```

Comment:
nitpick: in other places we use US english (serialise vs serialize). Furthermore @mhsdesign suggests to rename this:
```suggestion
    private static function serializeProjectedReferences(References $references): SerializedNodeReferences
```